src/components/navigation/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import styled from '@emotion/styled';
+import Menus from '../Menus/Menus';
+import WritePageMenus from '../WritePageMenus/WritePageMenus';
+
+const NavBar = () => {
+  const router = useRouter();
+
+  const resolvePosition = () => {
+    const pathname = router.pathname;
+    if (pathname == '/article/[id]') return 'position: absolute';
+    else if (pathname == '/write') return 'position: relative';
+    else return 'position:fixed';
```

Comment:
요것도 이렇게 할 수 있을 것 같아요

```suggestion
    if (pathname == '/article/[id]') return 'position: absolute';
    if (pathname == '/write') return 'position: relative';
    return 'position:fixed';
```

Reply:
오 넵! 감사합니다!